### PR TITLE
Update ml-pull-request.md.tmpl

### PR DESCRIPTION
**Pull request by yinxi-db**
update model registry description in `ml-pull-request.md.tmpl` to models in UC

### CODE REVIEW

File: template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl [hunk 1]
```diff
@@ -44,21 +44,17 @@ to address any test failures.
 {{ if (eq .input_include_feature_store `yes`) }}
 The integration test runs the feature engineering and model training notebooks as a multi-task Databricks Job in the staging workspace.
 It reads input data, performs feature transforms, and writes outputs to Feature Store tables in the staging workspace. 
-The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in the model registry. 
+The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in UC.
```
Comment (arpitjasa-db) [inline template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl:47]: Let's guard this with an if statement only if the models in UC option is selected

File: template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl [hunk 2]
```diff
@@ -44,21 +44,17 @@ to address any test failures.
...
+The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in UC. 
 The fitted model along with its metrics and params will also be logged to an MLflow run. 
 To debug failed integration test runs, click into the Databricks job run
-URL printed in the test logs. The job run page will contain a link to the MLflow model training run, which you can use
+URL printed in the test logs. The executed notebook of the job run will contain a link to the MLflow model training run. You can also use the Experiments page of the workspace
 to view training metrics or fetch and debug the model as needed.
 {{- else }}
 The integration test runs the model training notebook in the staging workspace, training, validating,
-and registering a new model version in the model registry. The fitted model along with its metrics and params
+and registering a new model version in UC. The fitted model along with its metrics and params
```
Comment (arpitjasa-db) [inline template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl:54]: ditto

File: template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl [hunk 3]
```diff
@@ -81,8 +77,8 @@ jobs in staging and production will automatically use your updated ML code.
 {{- end }}
 
 You can track the state of the ML pipelines for the current project from the MLflow registered model UI. Links:
-* [Staging workspace registered model]({{template `databricks_staging_workspace_host` .}}#mlflow/models/staging-{{template `model_name` .}})
-* [Prod workspace registered model]({{template `databricks_prod_workspace_host` .}}#mlflow/models/prod-{{template `model_name` .}})
+* [Staging model in UC]({{template `databricks_staging_workspace_host` .}}/explore/data/models/staging/{{template `project_name` .}}/{{template `model_name` .}})
```
Comment (arpitjasa-db) [inline template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl:80]: ditto

File: template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl [hunk 4]
```diff
@@ -44,21 +44,24 @@ to address any test failures.
 {{ if (eq .input_include_feature_store `yes`) }}
 The integration test runs the feature engineering and model training notebooks as a multi-task Databricks Job in the staging workspace.
 It reads input data, performs feature transforms, and writes outputs to Feature Store tables in the staging workspace. 
-The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in the model registry. 
+The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in 
+{{ if (eq .input_include_models_in_unity_catalog `no`) }} workspace model registry
```
Comment (arpitjasa-db) [inline template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl:48]:
```suggestion
{{ if (eq .input_include_models_in_unity_catalog `no`) }} the workspace model registry
```

File: template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl [hunk 5]
```diff
@@ -44,21 +44,24 @@ to address any test failures.
...
 The fitted model along with its metrics and params will also be logged to an MLflow run. 
 To debug failed integration test runs, click into the Databricks job run
-URL printed in the test logs. The job run page will contain a link to the MLflow model training run, which you can use
+URL printed in the test logs. The executed notebook of the job run will contain a link to the MLflow model training run. You can also use the Experiments page of the workspace
 to view training metrics or fetch and debug the model as needed.
 {{- else }}
 The integration test runs the model training notebook in the staging workspace, training, validating,
-and registering a new model version in the model registry. The fitted model along with its metrics and params
+and registering a new model version in 
+{{ if (eq .input_include_models_in_unity_catalog `no`) }} workspace model registry
```
Comment (arpitjasa-db) [inline template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl:58]:
```suggestion
{{ if (eq .input_include_models_in_unity_catalog `no`) }} the workspace model registry
```

File: template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl [hunk 6]
```diff
@@ -44,21 +44,24 @@ to address any test failures.
...
 It reads input data, performs feature transforms, and writes outputs to Feature Store tables in the staging workspace. 
-The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in the model registry. 
+The model training notebook uses these Feature Store tables as inputs to train, validate and register a new model version in 
+{{ if (eq .input_include_models_in_unity_catalog `no`) }} workspace model registry
+{{- else -}} UC
+{{end}}. 
 The fitted model along with its metrics and params will also be logged to an MLflow run. 
 To debug failed integration test runs, click into the Databricks job run
-URL printed in the test logs. The job run page will contain a link to the MLflow model training run, which you can use
+URL printed in the test logs. The executed notebook of the job run will contain a link to the MLflow model training run. You can also use the Experiments page of the workspace
```
Comment (arpitjasa-db) [inline template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl:53]:
```suggestion
URL printed in the test logs. The executed notebook of the job run will contain a link to the MLflow model training run, which you can use with the Experiments page in the workspace
```

File: template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl [hunk 7]
```diff
@@ -44,21 +44,24 @@ to address any test failures.
...
+{{end}}.
+The fitted model along with its metrics and params
 will also be logged to an MLflow run. To debug failed integration test runs, click into the Databricks job run
-URL printed in the test logs. The job run page will contain a link to the MLflow model training run:
-
-![Link to MLFlow Run](images/MLFlowRunLink.png)
-
-Click the MLflow run link to view training metrics or fetch and debug the model as needed.
-
+URL printed in the test logs. The executed notebook of the job run will contain a link to the MLflow model training run. You can also use the Experiments page of the workspace
```
Comment (arpitjasa-db) [inline template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl:63]:
```suggestion
URL printed in the test logs. The executed notebook of the job run will contain a link to the MLflow model training run. You can also use the Experiments page in the workspace
```

**Review by arpitjasa-db — COMMENTED** [review]
Minor nits

File: template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl [hunk 8]
```diff
@@ -44,21 +44,24 @@ to address any test failures.
...
+to view training metrics or fetch and debug the model as needed.
```
Comment (arpitjasa-db) [inline template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl:64]: Also if we're removing the link to the `MLFlowRunLink.png` might as well delete the file

Comment (yinxi-db) [reply template/{{.input_root_dir}}/docs/ml-pull-request.md.tmpl:64]: Done!

**Review by arpitjasa-db — APPROVED** [review]

**Pull request merged**